src/Integration/Aws/SimpleS3/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## NOT RELEASED
 
+## 2.1.0
```

Comment:
Hi @GrahamCampbell just to be sure I understood you properly, you mean the changelog should then look like so?
```md
# Change Log

## NOT RELEASED

### Changed

- Added optional `versionId` parameter to `SimpleS3Client::getPresignedUrl()`

## 2.0.0
...
```

And what about the change in `composer.json`, is it ok to just increase the minor version there like I did?

Reply:
The composer.lock change is correct. The branch alias indicates that the code is for the 2.1.x series. There need not be an actual 2.1.0 release yet.

Reply:
Yes, the change log should look like as in your comment.

Reply:
Thanks for the confirmation! One last thing, do you also confirm the "Changed" title? I am asking because @jderusse wrote something different here: https://github.com/async-aws/aws/pull/1626#discussion_r1426675749

Reply:
Yes, please make the change Jérémy asked for. He's actually maintainer here. :trollface:

Reply:
alright, done :+1: